book/src/continuous_integration/github_actions.md

```diff
@@ -1,7 +1,7 @@
 # GitHub Actions
 
-On the GitHub hosted runners, Clippy from the latest stable Rust version comes
-pre-installed. So all you have to do is to run `cargo clippy`.
+GitHub hosted runners' using the latest stable version of Rust has Clippy pre-installed.
```

Comment:
```suggestion
GitHub hosted runners using the latest stable version of Rust have Clippy pre-installed.
```
The apostrophe makes this possessive, there's also a tense mismatch with "using" and "has". :slightly_smiling_face:

Reply:
LOL where is my head! Thanks Serial!